### ISSUE TITLE
Fix sklearn warning for v1.0

### ISSUE THREAD

**Issue by adam2392** [mne-tools/mne-connectivity#44]
With sklearn v1.0, the following warning is issued for VAR model estimation.

```
 version 1.0 and will be removed in 1.2. Please leave the normalize parameter to its default value to silence this warning. The default behavior of this estimator is to not do any normalization. If normalization is needed please use sklearn.preprocessing.StandardScaler instead.
```

Just remove `normalize` kwarg.

**Comment by larsoner** [comment]
Already fixed in  https://github.com/mne-tools/mne-connectivity/commit/02818c9fa01949efec2ae3f2664ef0824c4d3212 as part of #27
